tasks/development.ts

```diff
@@ -208,4 +224,165 @@ export function load() {
+      await recordableFlipper.record.flip(1);
+
+      const receipts = await recorder.execute();
+      const { txParamsHash, txHash } = extractTxParamsHashAndTxHashFromSingleTransaction(receipts, "QueuedTransactionsBatch");
+      const { txParamsData } = await extractTimelockQueuedTransactionsBatchParamsDataAndHash(hre, txHash);
+
+      console.log("txHash:", txHash);
+      console.log("txParamsHash:", txParamsHash);
+      console.log("txParamsData:", txParamsData);
+    });
+
+  task("dev:approve-timelock-transaction-by-txParamsHash")
+    .addPositionalParam("timelockaddress")
+    .addPositionalParam("txParamsHash", "Transaction input data hash")
+    .setAction(async ({ timelockaddress, txParamsHash }, hre) => {
+      // this is dev only, normally done by multisig which is supposed to be the owner of the timelock
+
+      const timelockFactory = await hre.ethers.getContractFactory("Timelock");
+      const timelock = timelockFactory.attach(timelockaddress) as Timelock;
+
+      await (await timelock.approveTransactionsBatch(txParamsHash)).wait();
+    });
+
+  task("dev:approve-timelock-transaction-by-txHash")
+    .addPositionalParam("timelockaddress")
+    .addPositionalParam("txHash", "Transaction input hash")
+    .setAction(async ({ timelockaddress, txHash }, hre) => {
+      // this is dev only, normally done by multisig which is supposed to be the owner of the timelock
+
+      const { txParamsHash } = await extractTimelockQueuedTransactionsBatchParamsDataAndHash(hre, txHash);
+      console.log("txParamsHash", txParamsHash);
+
+      const timelockFactory = await hre.ethers.getContractFactory("Timelock");
+      const timelock = timelockFactory.attach(timelockaddress) as Timelock;
+
+      await (await timelock.approveTransactionsBatch(txParamsHash)).wait();
+    });
+
+  task("dev:approve-execute-timelock-transaction")
+    .addPositionalParam("timelockaddress")
+    .addPositionalParam("txRawParamsData")
+    .setAction(async ({ timelockaddress, txRawParamsData }, hre) => {
+      // this is dev only, normally done by multisig which is supposed to be the owner of the timelock
+
+      const timelockFactory = await hre.ethers.getContractFactory("Timelock");
+      const timelock = timelockFactory.attach(timelockaddress) as Timelock;
+
+      await (await timelock.approveAndExecuteTransactionsBatchRaw(txRawParamsData)).wait();
+    });
+
+  task("dev:flipper-show")
+    .addPositionalParam("flipperaddress")
+    .setAction(async ({ flipperaddress }, hre) => {
+      const flipperFactory = await hre.ethers.getContractFactory("Flipper");
+      const flipper = flipperFactory.attach(flipperaddress) as Flipper;
+
+      console.log("Owner:", await flipper.owner());
+      console.log("Flipper state[0] before:", await flipper.state(0));
+      console.log("Flipper state[1] before:", await flipper.state(1));
+      console.log("Flipper state[2] before:", await flipper.state(2));
+    });
+
+  task("timelock-show")
```

Comment:
```suggestion
  task("timelock:show")
```

Can we rename all the timelock tasks to start with `timelock:<action>:<specific>`?
`timelock:approve-and-execute:transaction`
`timelock:show`
`timelock:approve:by-txHash`
etc..

Also, can we move all the timelock tasks to a separate file?